target/linux/ramips/mt7620nand/target.mk

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (C) 2009 OpenWrt.org
+#
+
+SUBTARGET:=mt7620nand
+BOARDNAME:=MT7620 based boards with NAND flash
+FEATURES+=nand usb
+CPU_TYPE:=24kc
+
+DEFAULT_PACKAGES += kmod-rt2800-pci kmod-rt2800-soc kmod-mt76 wpad-mini
```

Comment:
Here you should select kmod-mt76x2 instead of kmod-mt76. That way you leave out kmod-mt7603, which you don't need.

Reply:
ok, Thank you for reminding